### ISSUE TITLE
Quill Issue.

### ISSUE THREAD

**Issue by Khorneflakes1** [Zenithrium/vanillaVoid#10]
The quasitemporal quill seems to activate at the same time as your final double jump is used up, so it ends up nullifying your final double jump. This can be seen with merc, heretic, and modded items that give double jumps. essentially meaning that the character wrongfully loses one of their jumps. This is particularly awful on merc, as it essentially removes his passive. Do you think you could add a short delay after your final jump is used up, or at least prevent quill from eating your upward momentum? I didn't log this as a bug report because I'm pretty sure it's not an error, just an oversight.

**Comment by Zenithrium** [comment]
yep, absolutely an oversight. would you believe me if i just totally forgot to test it on merc? 
i'll try fix it as soon as i can - got an exam tomorrow i very much need to study for so at the earliest ill try get it done tomorrow night
